github/ci/prow/templates/mirror-proxy.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: docker-mirror-proxy
+spec:
+  selector:
+    matchLabels:
+      app: docker-mirror-proxy
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: docker-mirror-proxy
+    spec:
+      terminationGracePeriodSeconds: 180
+      serviceAccountName: "caches"
+      nodeSelector:
+        type: bare-metal-external
+        ci.kubevirt.io/cachenode: "true"
+      tolerations:
+      - key: "cachenode"
+        operator: "Exists"
+        effect: "NoSchedule"
+      containers:
+      - name: mirror-proxy
+        image: fgimenez/docker-mirror-proxy:v0.1.0
+        env:
+        - name: ENABLE_MANIFEST_CACHE
+          value: "true"
+        volumeMounts:
+        - name: storage
+          mountPath: /docker_mirror_cache
+        - name: ca-cert
+          mountPath: /ca
+        ports:
+          - name: http
+            containerPort: 3128
+        resources:
+          requests:
+            memory: 3Gi
+          limits:
+            memory: 3Gi
+      volumes:
+      - name: ca-cert
+        secret:
+          secretName: docker-mirror-proxy
+      - name: storage
+        hostPath:
+          path: /var/lib/docker-mirror-proxy
```

Comment:
Can you use a sub-directory of where the docker mirror right now writes its data? We have there an extra partition just for this, so that we don't impact other applications if something goes wrong with the cache.

Reply:
Done the docker mirror is using `/var/lib/docker-mirror`, i've changed the proxy to use `/var/lib/docker-mirror/proxy`